Pipfile

```diff
@@ -46,6 +47,7 @@ requests = ">=2"
 whitenoise = { extras = ["brotli"], version = "==4.1.4" }
 db-to-sqlite = "*"
 sqlite-utils = "*"
+sqlalchemy = "~=1.4"
```

Comment:
there is a new version of sqlalchemy out, that breaks things. We pin the pipfile to avoid this heartache